app/models/training/question.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'content'
+
+module Training
+  class Question < Content
+    def page_type = nil
+
```

Comment:
I think this is a holdover from when I didn't have 'Page type' on questions - looks like that has been added, so this should probably go.

Reply:
That would be done on your ticket for fixing questions

Reply:
So you'll leave broken code until I can fix it?

Reply:
I can delete the line but the question specs will still fail for the time being. I thought it better to try and leave question related changes until that ticket got picked up. So yes I am in a way. 😉

Reply:
OK, it was just an observation - I can't remember if I added the new page_type on the question and video page content types. I had added these methods as stop gaps to get specs passing, in the hopes we could drive these changes through TDD, but we've gone a different way with your changes.